code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -141,6 +141,58 @@
 	/// Will be true if we lost target we were chasing
 	var/lost_target = FALSE
 
+	/// Cooldown for when to cleanup the ignore list.
+	COOLDOWN_DECLARE(ignore_list_cleanup_cd)
+
+/mob/living/simple_animal/bot/Initialize(mapload)
+	. = ..()
+	GLOB.bots_list += src
+	icon_living = icon_state
+	icon_dead = icon_state
+	access_card = new /obj/item/card/id(src)
```

Comment:
does this `new /obj/item/card/id(src)` needed to be deleted?

Reply:
No? It's the access card belonging to the bot.